homeassistant/components/network/__init__.py

```diff
@@ -0,0 +1,91 @@
+"""The Network Configuration integration."""
+from __future__ import annotations
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.components.websocket_api.connection import ActiveConnection
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import ConfigType
+from homeassistant.loader import bind_hass
+
+from .const import (
+    ATTR_ADAPTERS,
+    ATTR_CONFIGURED_ADAPTERS,
+    DOMAIN,
+    NETWORK_CONFIG_SCHEMA,
+)
+from .models import Adapter
+from .network import Network
+
+ZEROCONF_DOMAIN = "zeroconf"  # cannot import from zeroconf due to circular dep
+_LOGGER = logging.getLogger(__name__)
+
+
+@bind_hass
+async def async_get_adapters(hass: HomeAssistant) -> list[Adapter]:
```

Comment:
Is there a use case for knowing the configured adapters too from outside this integration?

I was reading the tests and if we would have access to both `network.adapters` and `network.configured_adapters` via helpers we could use those in the tests.

Reply:
I can't think of a use case currently.